podman/domain/manager.py

```diff
@@ -1,12 +1,13 @@
 """Base classes for PodmanResources and Manager's."""
 from abc import ABC, abstractmethod
 from collections import abc
-from typing import Any, List, Mapping, Optional, TypeVar, Union
+from typing import Any, Iterable, Mapping, Optional, TypeVar, Union
 
 from podman.api.client import APIClient
 
 # Methods use this Type when a subclass of PodmanResource is expected.
-PodmanResourceType: TypeVar = TypeVar("PodmanResourceType", bound="PodmanResource")
+# pylint: disable=invalid-name
+PodmanResourceType = TypeVar("PodmanResourceType", bound="PodmanResource")
```

Comment:
I'm not 100% confident in the `PodmanResource/PodmanResourceType` changes in this module.